xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -122,8 +124,8 @@
         <ReturnType>System.Diagnostics.ActivitySource</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the activity source associated with the activity.</summary>
```

Comment:
>Gets the activity source associated with the activity. [](start = 17, length = 54)

Get the activity source object create the Activity.

Reply:
```suggestion
        <summary>Gets the activity source that creates the activity.</summary>
```